### PR TITLE
fix: Update link to be ignored by the link checker

### PR DESCRIPTION
**Pull request by andrablaj**
# Description

Updates Google Drive link to be ignored by the link checker. [Discussion in Slack](https://medic.slack.com/archives/CUXB9SVT5/p1698789015037749?thread_ts=1698057359.974949&cid=CUXB9SVT5).

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: .github/scripts/muffet.sh [hunk 1]
```diff
@@ -40,5 +40,5 @@ muffet http://localhost:1313 \
   --exclude "https://tools.google.com.*" \
   --exclude "https://fhir.org/" \
   --exclude "https://docs.google.com/spreadsheets/d/12345ABCDEF/.*" \
-  --exclude "https://drive.google.com/file/d/1YPXoba9gVmD7SP-X88PpJIsIVGvY86_G/view?usp=share_link" \
+  --exclude "https://drive.google.com/file/d/1YPXoba9gVmD7SP-X88PpJIsIVGvY86_G*" \
```
Comment (mrjones-plip) [inline .github/scripts/muffet.sh:43]: I believe `muffet` expects a regex, so we need a `.` here:

```suggestion
  --exclude "https://drive.google.com/file/d/1YPXoba9gVmD7SP-X88PpJIsIVGvY86_G.*" \
```

**Review by mrjones-plip — CHANGES_REQUESTED** [review]
we're one char away from an approve!

**Comment by andrablaj** [comment]
Oh I went faster than the music @mrjones-plip! Thanks!

**Review by mrjones-plip — APPROVED** [review]
LGTM!

**Pull request merged**